assignment-client/src/octree/OctreeServer.cpp

```diff
@@ -1421,6 +1431,8 @@ void OctreeServer::sendStatsPacket() {
     statsArray1["4. persistFileLoadTime"] = getFileLoadTime();
     statsArray1["5. clients"] = getCurrentClientCount();
     statsArray1["6. threads"] = threadsStats;
+    statsArray1["uptime_seconds"] = getUptimeSeconds();
```

Comment:
the previous stats seem to be formatted differently,  perhaps format these the same way?

Reply:
What kind of differently you mean?

Reply:
6. threads
and the next ones you added do not lead with a number and also do not have the camelCasing

Reply:
Quite intentional. The "6. " stuff is a case of the code doing user presentation where it doesn't belong. Ideally all instances of this should be removed. Stat generating code really should just expose the data, formatting should be up to the UI.

Those additions duplicate existing data, but remove formatting. The data should be in a single, fixed unit (eg, 5500 ms), rather than formatting it as a human readable string ("5.5 seconds", "3 minutes, 10 seconds").

The underscore stuff is just because that's what Prometheus wants anyway, and DomainServerExporter actually converts "CamelCase" to "camel_case".